all-programs-summary-new/allprograms.json

```diff
@@ -0,0 +1,1718 @@
+[
+  {
+    "State": "AK",
+    "Title I 2018": 145776.0,
+    "Title I 2019": 124161.0,
+    "Title I 2020": 108899.0,
+    "Title I 2021": 0.0,
+    "Title I 2022": 0.0,
+    "Title I Total": 378836.0,
+    "Title II 2018": 7662394.39,
+    "Title II 2019": 6729781.45,
+    "Title II 2020": 8570223.77,
+    "Title II 2021": 9606129.47,
+    "Title II 2022": 8893772.0,
+    "Title II Total": 41462301.08,
+    "Crop Insurance 2018": 28332677.0,
+    "Crop Insurance 2019": 38519352.0,
+    "Crop Insurance 2020": 53144531.0,
+    "Crop Insurance 2021": 18255214.0,
+    "Crop Insurance 2022": 42701308.0,
+    "Crop Insurance Total": 180953082.0,
+    "SNAP 2018": 187357117.0,
+    "SNAP 2019": 171091672.0,
+    "SNAP 2020": 198298992.0,
+    "SNAP 2021": 288045258.0,
+    "SNAP 2022": 348717900.0,
+    "SNAP Total": 1193510939.0,
+    "2018 All Programs Total": 223497964.39,
+    "2019 All Programs Total": 216464966.45,
+    "2020 All Programs Total": 260122645.77,
+    "2021 All Programs Total": 315906601.47,
+    "2022 All Programs Total": 400312980.0,
+    "18-22 All Programs Total": 1416305158.08
+  },
+  {
+    "State": "AL",
+    "Title I 2018": 56548763.13,
+    "Title I 2019": 105364012.84,
+    "Title I 2020": 66207657.27,
+    "Title I 2021": 20372228.77,
+    "Title I 2022": 440010.11,
+    "Title I Total": 248932672.12,
+    "Title II 2018": 34646573.12,
+    "Title II 2019": 36603103.91,
+    "Title II 2020": 39102035.67,
+    "Title II 2021": 51295873.43,
+    "Title II 2022": 36045724.0,
+    "Title II Total": 197693310.13,
+    "Crop Insurance 2018": 81177.0,
```

Comment:
It looks like AL has the data for Alaska for crop insurance. 81,177 look like what we show now on dev